v12x/src/main/java/com/foursoft/harness/vec/v12x/navigations/DescriptionNavs.java

```diff
@@ -86,9 +87,11 @@ public static Function<List<? extends VecAbstractLocalizedString>, Optional<Stri
             }
 
             return localizedStrings.stream()
+                    .filter(Objects::nonNull)
                     .filter(d -> !(d instanceof VecLocalizedTypedString))
                     .filter(VecPredicates.isLanguageCode(vecLanguageCode))
```

Comment:
Is this predicate also null safe?

Reply:
Yes, `Objects.nonNull` just executes `obj != null`.